AllASPFuncs.py

```diff
@@ -1255,3 +1264,18 @@ def banner(text):
     length = len(text) + 8
     message = "\n" + "#" * length +'\n'+'#   ' + text + '   # \n'+ "#" * length
     Lager.debug(message)
+
+def get_SED_list(ID, exposures, fetch_SED, object_type, sn_path):
```

Comment:
Add doc string explaining what function does, what it takes as input, and what it produces as output.